### DISCUSSION TITLE
編集画面が異常に重い

### DISCUSSION THREAD

**Discussion by kottaro123456** [increments/qiita-discussions#947] [要望・アイデア・不具合]
編集画面だけが異常に重くなります。

新規記事画面では普通に入力できます。
しかし、編集画面だけとても重く、文字を打って3秒後ぐらいにやっと出てくるくらいなのでまともに文章を書けません。
変換はリアルタイムに出てきて、この録画もできているのでスペックが足りない等の問題ではないと考えます。
変換確定する事にエラーが出てきて、編集画面のページだけ動作が止まります。
ただし、追加で文章を書いたり（画面には出てきませんが）他のタブやアプリに行ったりすることはできます。

WindowsのChrome, Edge, Firefox、AndroidのChrome,Edge、iPhoneのSafariで試してみましたが、全て動作が安定していませんでした。
家のWifi以外にもモバイル回線(楽天)でも発生します。
特に広告ブロック等は導入していません。

ご確認していただければ幸いです。

https://github.com/user-attachments/assets/7eda0331-1b59-482f-bef4-62d29682205e

![image](https://github.com/user-attachments/assets/a573c4b3-c351-45ca-a838-a83b4a26bea8)

**Comment by P-SiZK** [comment]
@MiyakoCam 
ご報告ありがとうございます！

一つのコードブロックに記述されるコードの量が多い場合かつ、syntax highlight が on の場合、重くなってしまう問題があります。
改善を検討していますが、修正されるまでは以下のいずれかの対応をしていただけると幸いです。

- コードブロックの指定言語を `plaintext` 等にして syntax highlight を off にする
- [GitHub Gist](https://qiita.com/Qiita/items/612e2e149b9f9451c144#github-gist) などを使って記事にコードを埋め込む

ご不便をおかけいたしますが、何卒よろしくお願いいたします :bow:

**Comment by kottaro123456** [comment]
@P-SiZK 様
ご確認とご返信ありがとうございます。

確かにこの後に3つのファイルをまるまるコードとしてコピペしているので大量にハイライトしていたのだと思います。

また、コードの埋め込みなどの対応策もありがとうございます。
Github GistやCodepenなどでの埋め込みができるんですね！~~(初めて知りました…)~~
